test/05-supervisor-release.ts

```diff
@@ -51,4 +53,74 @@ describe('supervisor release', function() {
 			.delete(`/resin/supervisor_release(${supervisorReleaseId})`)
 			.expect(200);
 	});
+
+	describe('Devices running supervisor releases', () => {
```

Comment:
Is there any reason for this to be nested? I'd rather it be a separate top-level describe

Reply:
Only that it's directly related to supervisor releases, which is what the above describe is named

Reply:
Could you un-nest it please and rename the other describe to `supervisor release management` as that's more accurate to what it is